pkg/agent/client/conn.go

```diff
@@ -50,6 +56,7 @@ func (c *conn) Write(data []byte) (n int, err error) {
 	}
 
 	klog.Infof("[tracing] send req %+v", req)
+	fmt.Printf("[tracing] send req %+v\n", req)
```

Comment:
Seems like we should be consistently using klog. If you do not direct the output to a file it should be going to stdout.